diktat-maven-plugin/pom.xml

```diff
@@ -165,6 +192,13 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>me.fabriciorby</groupId>
+                        <artifactId>maven-surefire-junit5-tree-reporter</artifactId>
+                        <version>1.1.0</version>
```

Comment:
can you please expect this property to a common variable

Reply:
`Thanks, addressed with ea57ecc084851a8006383a4fce39a0dd18fa283d.